testing/tests/DevExpress.ui.widgets.editors/textEditorParts/mask.tests.js

```diff
@@ -320,6 +322,22 @@ QUnit.module('typing', moduleConfig, () => {
         assert.equal(keyboard.caret().start, 3, 'caret in correct position');
     });
 
+    QUnit.test('autofill input with mask should not throw error (T996580)', function(assert) {
+        const $textEditor = $('#texteditor').dxTextEditor({
+            mask: '+1 (X00) 000-0000',
+            value: '+1'
```

Comment:
Is it necessary?

Reply:
Yes, error is only logged when mask enabled. value is also necessary because `_inputHandler` has a return statement when `!currentCaret.end`